core/src/main/java/org/mule/runtime/core/internal/processor/LoggerMessageProcessor.java

```diff
@@ -32,13 +34,15 @@
 /**
  * MessageProcessor implementation that logs the current element of a value evaluated from it using an expression evaluator. By
  * default the current messages is logged using the {@link Level#INFO} level to the
- * 'org.mule.runtime.core.internal.processor.LoggerMessageProcessor' category. The level and category can both be configured to suit
- * your needs.
+ * 'org.mule.runtime.core.internal.processor.LoggerMessageProcessor' category. The level and category can both be configured to
+ * suit your needs.
  */
 public class LoggerMessageProcessor extends AbstractComponent implements Processor, Initialisable, MuleContextAware {
 
+  public static final String MULE_LOGGING_BLOCKING_CATEGORIES = SYSTEM_PROPERTY_PREFIX + "logging.blockingCategories";
```

Comment:
move this to MuleSystemProperties in mule-api with the proper documentation, since this property is intended for users to tune/configure